SimPEG/Directives.py

```diff
@@ -288,38 +288,85 @@ class SaveOutputEveryIteration(SaveEveryIteration):
     """SaveModelEveryIteration"""
 
     header = None
+    save_txt = True
+
+    beta = None
+    phi_d = None
+    phi_m = None
+    phi_m_small = None
+    phi_m_smooth_x = None
+    phi_m_smooth_y = None
+    phi_m_smooth_z = None
+    phi = None
 
     def initialize(self):
-        print("SimPEG.SaveOutputEveryIteration will save your inversion progress as: '###-{0!s}.txt'".format(self.fileName))
-        f = open(self.fileName+'.txt', 'w')
-        self.header = "  #     beta     phi_d     phi_m   phi_m_small     phi_m_smoomth_x     phi_m_smoomth_y     phi_m_smoomth_z      f\n"
-        f.write(self.header)
-        f.close()
+        if self.save_txt is True:
+            print(
+                "SimPEG.SaveOutputEveryIteration will save your inversion "
+                "progress as: '###-{0!s}.txt'".format(self.fileName)
+            )
+            f = open(self.fileName+'.txt', 'w')
+            self.header = "  #     beta     phi_d     phi_m   phi_m_small     phi_m_smoomth_x     phi_m_smoomth_y     phi_m_smoomth_z      phi\n"
```

Comment:
it can be computed pretty easily just by adding phi_m_smooth_x, ... do you think it is worth saving separately? we could write it as a property of the directive instead (eg. `save.phi_m_smooth` just adds the components together? )